HCK14_2023_Janelia/README.md

```diff
@@ -0,0 +1,180 @@
+# NWB User Days and Developer Hackathon 2023
```

Comment:
Can you please rename the folder to ``HCK15_2023_Janelia`` (i.e., 15 instead of 14)

Reply:
I just made that change. I'll add a few more edits now and then I think this is good to merge.